transforms/no-implicit-this/index.js

```diff
@@ -1,12 +1,10 @@
 const path = require('path');
 const fs = require('fs');
 
-const { parse: parseHbs, print: printHbs } = require('ember-template-recast');
-const { determineThisUsage } = require('./helpers/determine-this-usage');
+const recast = require('ember-template-recast');
+const transformPlugin = require('./helpers/plugin');
```

Comment:
a much needed cognitive change! :100: 

the previous nomenclature confused me every time I looked at it. ha